pages/_app.page.tsx

```diff
@@ -97,6 +97,12 @@ const App = ({
       },
     },
     enabled: !!process.env.ROLLBAR_ACCESS_TOKEN,
+    checkIgnore: (_, args) => {
+      // Ignore React hydration warnings
+      return (
```

Comment:
Can we specifically just ignore React errors 418, 423, and 425? I wouldn't want us to miss other React errors.